tests/serde.rs

```diff
@@ -541,6 +541,25 @@ fn maps_objects_string_key() {
     test_via_json_with_config(src, Serializer::new().serialize_maps_as_objects(true));
 }
 
+#[wasm_bindgen_test]
+fn serialize_json_compatible() {
+    #[derive(Debug, Serialize, Deserialize, PartialEq, Eq)]
+    struct Struct {
+        a: HashMap<String, ()>,
+        b: Option<i32>,
+    }
+
+    let serializer = Serializer::json_compatible();
+    let x = Struct {
+        a: hashmap! {
+            "foo".to_string() => (),
+            "bar".to_string() => (),
+        },
+        b: None,
+    };
+    test_via_json_with_config(x, serializer);
```

Comment:
I'd probably just inline it to make config visible right alongside the call

```suggestion
    test_via_json_with_config(x, Serializer::json_compatible());
```